lib/launcher.rb

```diff
@@ -40,6 +40,9 @@ def https_app
           ::Proxy::Plugins.enabled_plugins.each {|p| instance_eval(p.https_rackup)}
         end
 
+        ssl_options = OpenSSL::SSL::SSLContext::DEFAULT_PARAMS[:options]
```

Comment:
Where are you seeing compression on by default? It was patched to be disabled as part of #9424 in 1.9.3 and newer. It also appears that RH has patched el6 to disable it as well.